server/views/components/find-us/index.njk

```diff
@@ -0,0 +1,11 @@
+<div class="find-us">
+  {% icon 'other/location' %}
+  <p class="find-us__address" itemprop="address" itemscope itemtype="https://schema.org/PostalAddress">
```

Comment:
What do you think - we could stick the LD in here - or we could have one large JSON-LD at the top of the page somewhere... I'm not really sure.